test/spec/controllers/foyer/ressources/ressources.js

```diff
@@ -276,7 +262,7 @@ describe('Controller: FoyerRessourcesCtrl', function() {
             expect(individu.interruptedRessources[0]).toEqual('test2');
         });
 
-        it('devrait sauvegarder les "autres revenus professionnels" directement sur l’individu', function() {
+        it('devrait sauvegarder les "autres revenus professionnels" dans les ressources', function() {
```

Comment:
English consistency would be welcome.

Not mandatory here, though, but it would follow the [boy-scout rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)  :wink: